### PR TITLE
Add missing models/endpoints for schemas API.

### PR DESCRIPTION
**Pull request by laura-rodriguez**
Add missing endpoints and models for [Schemas API](https://developer.okta.com/docs/reference/api/schemas)

See the [generated code + UTs and ITs in .NET](https://github.com/okta/okta-sdk-dotnet/pull/477) to get a sense of what's included 

**New Operations:**
* GetUserSchema
* UpdateUserSchema
* GetApplicationUserSchema
* UpdateApplicationUserSchema

Fix
* OKTA-347834
* https://github.com/okta/okta-sdk-golang/issues/188

### CODE REVIEW

File: resources/spec.yaml [hunk 1]
```diff
@@ -3959,6 +3959,97 @@ paths:
...
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/UserSchema"
+      security:
+        - api_token: []
+      summary:  "Partial updates on the User Profile properties of the Application User Schema."
+      tags:
+        - UserSchema
+  /api/v1/meta/schemas/user/{schemaId}:
```
[the remark this answers is not in the document]
Comment (laura-rodriguez) [reply resources/spec.yaml:4006]: https://oktainc.atlassian.net/browse/OKTA-373555

File: resources/spec.yaml [hunk 2]
```diff
@@ -3959,6 +3959,97 @@ paths:
...
+    get:
+      description: Fetches the schema for a Schema Id.
+      operationId: getUserSchema
+      parameters:
+        - in: path
```
[the remark this answers is not in the document]
Comment (laura-rodriguez) [reply resources/spec.yaml:4011]: https://oktainc.atlassian.net/browse/OKTA-373555

File: resources/spec.yaml [hunk 3]
```diff
@@ -9194,6 +9285,189 @@ definitions:
     type: object
     x-okta-tags:
       - Policy
+  UserSchema:
+    type: "object"
+    properties:
+      id:
+        type: "string"
+      $schema:
```
[the remark this answers is not in the document]
Comment (laura-rodriguez) [reply resources/spec.yaml:9293]: I'll make sure to add a description in the changelog.

File: resources/spec.yaml [hunk 4]
```diff
@@ -9194,6 +9285,189 @@ definitions:
...
+        type: "string"
+      definitions:  
+        $ref: "#/definitions/UserSchemaDefinitions"
+      type:
+        type: "string"
+      properties:
+        additionalProperties:
+          type: object
+        type: object
+      _links:
```
[the remark this answers is not in the document]
Comment (laura-rodriguez) [reply resources/spec.yaml:9313]: https://oktainc.atlassian.net/browse/OKTA-373555

**Review by bretterer — APPROVED** [review]

**Pull request merged**